hazelcast-spring/src/main/resources/hazelcast-spring-5.0.xsd

```diff
@@ -4420,7 +4434,7 @@
         <xs:attribute name="data-load-timeout-seconds" type="parameterized-unsigned-int">
             <xs:annotation>
                 <xs:documentation>
-                    Data load timeout for hot-restart process,
+                    DEPRECATED. Data load timeout for hot-restart process,
```

Comment:
If it is deprecated I'd add a comment what to use instead.

Reply:
it is from the [Slack](https://hazelcast.slack.com/archives/C01PD4KN9FB/p1624960363143500) discussion. Basically, the motivation was that `data-load-timeout-seconds` attribute is tied into current Hot Restart, but in the future it may not be applicable. I will add comment telling users to leave it to default value.